### DISCUSSION TITLE
posts/2024/11/sass-mixed-declarations-change/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#58] [Announcements]
# posts/2024/11/sass-mixed-declarations-change/

If you upgrade to Dart Sass 1.77.7+, you may need to move around some nested items in your styling.

https://www.brycewray.com/posts/2024/11/sass-mixed-declarations-change/

**Comment by davidsneighbour** [comment]
> Thus, if you’re a Hugo/Sass user who hasn’t upgraded Embedded Dart Sass recently, you might want to make time for testing your existing SCSS with a post-v.1.77.7 version.

Or, you look into linting. I was wondering why Stylelint was complaining about this for years. They knew... THEY KNEW!!!!!1One
